lib/db/mem.js

```diff
@@ -169,10 +169,6 @@ module.exports = function (log, error) {
   Memory.prototype.createAccountResetToken = function (tokenId, accountResetToken) {
     tokenId = tokenId.toString('hex')
 
-    if ( accountResetTokens[tokenId] ) {
```

Comment:
The comments don't mention the reason for this but presumably it was just some errant rejection that wasn't being picked up by the pre-promisified tests? Is there a case for an explicit assertion anywhere, or is it sufficient that the promise simply doesn't reject?